### PR TITLE
Updates to TR-3 CT-16 & TR-5

### PR DESCRIPTION
**Pull request by aateeque**

### CODE REVIEW

**Comment by ColinEberhardt** [comment]
Thanks for another great contribution @aateeque 

For [TR-5: Instability in foundation model behaviour](https://github.com/aateeque/ai-readiness/blob/main/docs/_threats/tr-5.md) would it be possible to make it clearer that there are at least two sources of instability:
- Non-determinism - LLMs are probabilistic, which means given the same prompt, the output may differ between inferences.
- Versioning - This is where the underlying weights or architecture of the LLMs has changed. While this is more deterministic, some third-party hosted LLMs make these changes without notifying the end-user, which makes this a close relative to TR-11

**Review by ColinEberhardt — CHANGES_REQUESTED** [review]
as per in-line comment

File: docs/_threats/tr-5.md [hunk 1]
```diff
@@ -10,7 +10,18 @@ external-refs:
 
 ---
 
-The foundation model behaviour may change over time if the third-party doesn't have rigorous version control (covered in Threat 11), or silently changes model versions without notifying the consumer.
+Instability in foundation model behaviour would manifest itself as deviation in prompt output from previous tests.
```
[the remark this answers is not in the document]
Comment (aateeque) [reply docs/_threats/tr-5.md:13]: Thanks for the great suggestion @ColinEberhardt; incorporarted

**Comment by aateeque** [comment]
Thanks @ColinEberhardt - changes incorporated

**Comment by lucaborella89** [comment]
@aateeque @ColinEberhardt as per MS review (https://github.com/finos/ai-readiness/issues/79), TR5 could be merged with TR6

**Pull request merged**